service/history/queues/executable.go

```diff
@@ -155,7 +155,7 @@ func NewStateStaleError(message string) StaleStateError {
 }
 
 func (e StaleStateError) Error() string {
-	return "task stale: " + e.Message
+	return "state stale: " + e.Message
```

Comment:
Glad I caught this before it was deployed, could have made for some confusing debugging.